vnext/Microsoft.ReactNative/Modules/AppThemeModuleUwp.cpp

```diff
@@ -73,9 +74,7 @@ std::string AppTheme::formatRGB(winrt::Windows::UI::Color ElementColor) {
 }
 
 void AppTheme::fireEvent(std::string const &eventName, folly::dynamic &&eventData) {
-  if (auto instance = m_wkReactInstance.lock()) {
-    instance->CallJsFunction("RCTDeviceEventEmitter", "emit", folly::dynamic::array(eventName, std::move(eventData)));
-  }
+  m_context->CallJSFunction("RCTDeviceEventEmitter", "emit", folly::dynamic::array(eventName, std::move(eventData)));
```

Comment:
Just for my own understanding, does the context internally keep week ownership?

Reply:
Yes.  Context is basically a weak pointer on an instance.